lib/Cake/View/Helper.php

```diff
@@ -958,7 +958,7 @@ protected function _reset() {
  * @return void
  */
 	protected function _clean() {
-		if (get_magic_quotes_gpc()) {
+		if (function_exists('get_magic_quotes_gpc') && get_magic_quotes_gpc()) {
```

Comment:
I think we can remove this and keep only `else` part:
```
$this->_cleaned = $this->_tainted;
```

Reply:
That's true. Changed: https://github.com/kamilwylegala/cakephp2-php8/pull/23/commits/bb4e190